lib/jazzy/sourcekitten.rb

```diff
@@ -52,8 +52,33 @@ def self.make_doc_urls(docs, parents)
       end
     end
 
+    def self.assert_xcode_location
+      expected_xcode_select_path =
+        Pathname('/Applications/Xcode.app/Contents/Developer')
+      return if xcode_developer_directory == expected_xcode_select_path
+      raise 'Please install Xcode 6.1 or 6.2 in ' \
```

Comment:
Great :+1: this will probably be sufficient (no need for more, but at least it will be mentioned)
```diff
- raise 'Please install Xcode 6.1 or 6.2 in ' \
+ raise 'Please install or symlink Xcode 6.1 or 6.2 in ' \
```